### DISCUSSION TITLE
wire entangle + debounce

### DISCUSSION THREAD

**Discussion by usernotnull** [alpinejs/alpine#2584] [3. Feature ideas]
Currently, we can defer a livewire entangled property:

```
$wire.entangle('foo').defer
```

How about adding a .debounce also? My use case is in a multi-tag select component. I wanted to add a 500ms debounce if the user removes tags quickly. The only way around it is adding an update button and defer, but debouncing is much cleaner.

**Comment by SimoTod** [comment]
FYI, $wire and entangle are defined in livewire and are not part of the Alpine core

**Reply by usernotnull** [reply]
Thanks for the info. I'll move the idea there.
